src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -285,12 +285,9 @@ private static void CaptureAnnotatedCodeLocations(IEnumerable<AnnotatedCodeLocat
             }
         }
 
-        private static bool IsError(ResultKind kind)
+        private static bool IsError(ResultLevel level)
         {
-            return 
-                kind == ResultKind.ConfigurationError ||
-                kind == ResultKind.Error ||
-                kind == ResultKind.InternalError;
+            return level == ResultLevel.Error;
```

Comment:
Filed [https://github.com/Microsoft/sarif-sdk/issues/230](https://github.com/Microsoft/sarif-sdk/issues/230).

---
In reply to: [61639020](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61639020) [](ancestors = 61639020)